docs/validator/keys.md

```diff
@@ -54,6 +54,10 @@ If validator keys are changed _during an epoch in which the validator is staking
 
 For concrete examples of keys being used as identifiers, you can see a list of validators and active nodes on various NEAR networks here:
 
-- NEAR TestNet
+- NEAR TestNet (staking currently disabled)
```

Comment:
please update these URL, we just merged a PR that moves `nearprotocol.com` to `near.org`